### DISCUSSION TITLE
PatchTool 6.28 12/22

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#251] [Announcements]
漢化更新至中國服6.2，同時校正了大部分的翻譯問題。

修正 #242 #243 #247 #248 #249 #250

檔案：

- FFXIVChnTextPatch-20221222.rar：本體。一般而言，下載這個即可。JRE縮小過。

選用：

- rawexd-zhcn-20221222.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- [FFXIV-jar-0211.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.56/FFXIV-jar-0211.rar)：如果你有遇到JNI error，請參考 https://github.com/GpointChen/FFXIVChnTextPatch-GP/issues/90 放到與exe同位置使用。
- [backup-20221102.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.70.2/backup-20221102.rar)：乾淨的0X0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。

目前測試，在之前的更新後，第三方4k字體包可能會導致遊戲崩潰。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v2.71.0'>PatchTool 6.28 12/22</a>.</em>

**Comment by Manbbb** [comment]
感謝大大，更新辛苦了^_^
無人島衝啊！

**Reply by Manbbb** [reply]
報告 <^_^ 無人島耕種小員對話內容還沒有翻譯
![ScreenHunter 578](https://user-images.githubusercontent.com/103889052/209205778-f357c513-e6cc-475f-8f36-102d72ce350b.jpg)

![image](https://user-images.githubusercontent.com/103889052/209206144-bf686021-8828-49e8-8c10-c04505acdf05.png)

**Comment by KasunaStony** [comment]
辛苦啦！请问简中零式有些文本还没有翻译是因为国服零式还没开吗？谢谢！

**Reply by GpointChen** [reply]
零式應該開了吧，有哪裡還沒翻譯？可以給個截圖之類的，我回頭確認一下。

**Reply by cream313** [reply]
> 零式應該開了吧，有哪裡還沒翻譯？可以給個截圖之類的，我回頭確認一下。

簡中 零式有一些沒中文 一開始的分散/分攤 都沒中文 還是日文

**Reply by cream313** [reply]
1-4層 都各有一些沒中文

**Comment by okami2223** [comment]
感謝作者更新
另外可以附上本次的復原backup檔案嗎？
再麻煩您了!

**Reply by GpointChen** [reply]
這次的復原檔案和上次應該是一樣的（國際服沒有更新），你可以點bckup那個連結下載。

**Reply by okami2223** [reply]
謝謝你！

**Comment by Zhong0514** [comment]
抱歉第一次用 點了漢化以後跑到一半說程式錯誤該怎麼辦?

**Reply by GpointChen** [reply]
最常見的原因是原本的遊戲檔案有損毀導致無法覆蓋漢化，你可以查看log檔案確認是停在哪一步。

你可以下載[backup-20221102.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.70.2/backup-20221102.rar)，解壓後把裡面的000000和0a0000等六個檔案覆蓋遊戲資料夾內的同名檔案（應該會位於`~\SquareEnix\FINAL FANTASY XIV - A Realm Reborn\game\sqpack\ffxiv`），然後再次嘗試漢化。

在覆蓋前建議先備份會被覆蓋的檔案。

**Reply by Zhong0514** [reply]
好的 已經可以用了 謝謝您

**Comment by xxcornss** [comment]
![image](https://user-images.githubusercontent.com/122058757/210887468-d3618e1b-801e-4528-8769-60fcfc13396b.png)
之前都沒有事的 是這一次更新檔的exe會這樣 已經有從java更新到18
![image](https://user-images.githubusercontent.com/122058757/210887608-b429c833-b375-4811-a1c0-d88c4d126503.png)

**Reply by GpointChen** [reply]
Exe本身沒有做任何更動，如果之前的可以使用，你可以試著將新版本中的`resource/rawexd`資料夾覆蓋到舊版後再執行漢化。

**Reply by xxcornss** [reply]
另外岩岩打8樓 隻boss講野會淨係得個對話泡泡 入面係冇字

**Comment by Bos-hunting** [comment]
今天國際服會有6.3的更新，希望更新後會有兼容當前漢化的漢化包，麻煩了感恩

**Reply by msz006zeta** [reply]
回報一下，角色走到庫爾扎斯這一帶就會error歐  :)
![image](https://user-images.githubusercontent.com/122380029/211589560-7b036733-d3ac-49d7-b5d1-af9efed47cf2.png)
![image](https://user-images.githubusercontent.com/122380029/211590055-17d57d29-bdb4-4926-9549-d908f3bb6d7e.png)

**Reply by okami2223** [reply]
今天剛更新而已，等等作者修正吧

**Reply by walley06200** [reply]
![未命名](https://user-images.githubusercontent.com/122627238/212360093-94d06a83-3685-4af3-bc69-848ee7915db9.png)
回報一下，5.0謊言的泡沫以後也會跳出類似的訊息

**Reply by GpointChen** [reply]
你是用最新的版本嗎？6.28的漢化會有問題是正常的。
